libraries/omnibus_service_resource.rb

```diff
@@ -24,6 +24,7 @@ class OmnibusService < Chef::Resource::LWRPBase
 
       attribute :ctl_command, kind_of: String
       attribute :service_name, kind_of: String, regex: %r{[\w-]+/[\w-]+}, name_attribute: true
+      attribute :version, kind_of: String, default: '0.0.0'
```

Comment:
`omnibus_service` uses the `chef_ctl_command` method to get the ctl command for the product, and it takes version as an argument. I can do something like this:

```ruby
    def chef_ctl_command(product)
      version = new_resource.respond_to?(:version) ? new_resource.version : '0.0.0'
      product_lookup(product, version)['ctl-command']
    end
```